src/libopenrave/planner.cpp

```diff
@@ -217,7 +217,17 @@ void PlannerStatus::InitCollisionReport(CollisionReportPtr& newreport)
     }
 }
 
-void PlannerStatus::AddCollisionReport(const CollisionReport& collisionReport)
+void PlannerStatus::InitWorkspaceTraj(TrajectoryBaseConstPtr& newworkspacetraj)
```

Comment:
`const TrajectoryBaseConstPtr& newworkspacetraj`

Reply:
1. It should be called `SetWorkspaceTraj`
2. Actually why have this function? Since the member is already public.